acsoo/templates/project/+project.name+/README.rst.bob

```diff
@@ -1,3 +1,9 @@
+{{% if odoo.series in ['11.0'] %}}
+{{% set mk_venv_extra = '--python=/usr/bin/python3' %}}
```

Comment:
`$(which python3)` instead of `/usr/bin/python`

`$(which python)` for python2, se in case the default python for virtualenv changes we are safe

Reply:
BTW, the python version could be set in `pre_render_project` in `hook.py`, since we'll need it a bit everywhere.

There are examples in bobtemplates.odoo.

Reply:
About ```$(which python3)```, are you sure we can use it here? Maybe it's a bit weird, but we still can create an Odoo 11 project using python2.7, right?

Reply:
My comment has no sense ^^

Reply:
I updated my code but I'm still not sure about using "which", because for example if I use ```acsoo``` in a virtualenv, ```which``` will return the virtualenv's python file path, and that's not the python executable I want to use.

Reply:
My last comment should clarify this: in the places where we are going to use this $(which), we are not in an activated virtualenv.